cmd/notification.go

```diff
@@ -163,9 +158,10 @@ func (sys *NotificationSys) ReloadFormat(dryRun bool) []NotificationPeerErr {
 // LoadUsers - calls LoadUsers RPC call on all peers.
 func (sys *NotificationSys) LoadUsers() []NotificationPeerErr {
 	var idx = 0
-	ng := WithNPeers(len(sys.peerRPCClientMap))
-	for addr, client := range sys.peerRPCClientMap {
-		ng.Go(context.Background(), client.LoadUsers, idx, addr)
+	ng := WithNPeers(len(sys.peerHosts))
+	for _, addr := range sys.peerHosts {
```

Comment:
We don't need idx anymore it was used with map previously, since now we have peerHosts as slice we can simply do 

```
for idx, addr := range sys.peerHosts {
```

Reply:
Have gone back to using a `map` since @krishnasrinivas wanted to have a cached copy of `client` that corresponds to a `peer`